src/fmpz/link/fmpz_gc.c

```diff
@@ -77,8 +97,21 @@ void _fmpz_clear_mpz(fmpz f)
 {
     __mpz_struct * ptr = COEFF_TO_PTR(f);
 
-    if (ptr->_mp_alloc > FLINT_MPZ_MAX_CACHE_LIMBS)
-        mpz_realloc2(ptr, 1);
+    if (ALLOC(ptr) > FLINT_MPZ_MAX_CACHE_LIMBS)
+    {
+        /* mpz_realloc2(ptr, 1); */
```

Comment:
The comment doesn't seem to match the code.

Reply:
No, sorry, should've mentioned this. I think it is good if set a lowest allowed number of limbs allocated.

The number I think fits the best here is two. Why? Well, consider the basic arithmetic operations (addition, subtraction, multiplication and division) of small fmpzs. The most number of limbs that is required to store the result is two. Hence, I believe this is the best number (or perhaps three).

With the comment, I just wanted to be clear what I replaced it with, sorry for the confusion.